src/Core/Grid/Definition/Factory/CustomerGroupsGridDefinitionFactory.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Grid\Definition\Factory;
+
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\BulkActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\DataColumn;
+
+class CustomerGroupsGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    public const GRID_ID = 'customer_groups';
+
+    protected function getId()
```

Comment:
add "{@inheritDoc}" annotation to every method that inherits parent return type and params.
The old interface wasn't typed, so we at least add those inheritdocs for that matter.